lib/child-writer/index.js

```diff
@@ -126,8 +126,28 @@ exports.write = (image, drive, options) => {
    * }));
    */
   const bridgeRobotMessage = (message) => {
+    const ROBOT_COMMAND_LOG = 'log';
```

Comment:
Hmmm, why add a constant for `ROBOT_COMMAND_LOG` but not also a constant for e.g. `ROBOT_COMMAND_ERROR` ?